addons/mail/static/src/less/thread.less

```diff
@@ -69,23 +69,24 @@
                 opacity: 0;
             }
         }
-
-
-        i.o_thread_message_star, i.o_thread_message_needaction, i.o_thread_message_reply, i.o_thread_message_email  {
+        .o_thread_icons {
             cursor: pointer;
-            opacity: 0;
-            &.fa-star {
+            .fa-star {
                 opacity: @mail-thread-icon-opacity;
                 color: gold;
             }
+            :not(.fa-star) {
+                opacity: 0;
+            }
+            &.touch_screen > * {
+                opacity: @mail-thread-icon-opacity;
+            }
```

Comment:
So this should become something like: 
```less
.o_web_client.o_touch_device {
    .your_selector {
        ...
    }
}
```
Also, always start your class names with a `o_`